### PR TITLE
systemd-path: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-path.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# systemd-path
+
+> List and query system and user paths.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-path.html>.
+
+- When invoked without arguments, the command displays a list of known paths and their current values:
```
Comment (acuteenvy) [inline pages/linux/systemd-path.md:6]:
```suggestion
- Display a list of known paths and their current values:
```

File: pages/linux/systemd-path.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`systemd-path`
+
+- When at least one argument is provided, the path with the specified name is queried, and its value is displayed:
```
Comment (acuteenvy) [inline pages/linux/systemd-path.md:10]:
```suggestion
- Query the specified path and display its value:
```

File: pages/linux/systemd-path.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`systemd-path "{{query_name}}"`
```
Comment (acuteenvy) [inline pages/linux/systemd-path.md:12]:
```suggestion
`systemd-path "{{path_name}}"`
```

File: pages/linux/systemd-path.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print paths with the specified string as a suffix:
+
+`systemd-path --suffix="{{suffix_string}}"`
```
Comment (acuteenvy) [inline pages/linux/systemd-path.md:16]:
```suggestion
- Suffix printed paths with `suffix_string`:

`systemd-path --suffix {{suffix_string}}`
```

Comment (quantumflo) [reply pages/linux/systemd-path.md]: @acuteenvy updated

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**